Assets/Scripts/UIController.cs

```diff
@@ -36,13 +40,15 @@ void FixedUpdate()
                     text.text = string.Empty;
                 }
             }
-            if (Mouse.current.leftButton.isPressed || Mouse.current.rightButton.isPressed)
-            {
-                var position = Mouse.current.position.ReadValue();
-                pointer.rectTransform.anchoredPosition = position;
-                pointer.color = Color.red;
-            }
-            else if (UnityEngine.InputSystem.EnhancedTouch.Touch.activeTouches.Count > 0)
+
+            bool pointerFromMouse
+                =  HighlightPointerFromMouse(RemoteInput.RemoteMouse, m_renderStreaming.GetStreamingSize())
```

Comment:
TODO:: we wouldn't like to depend on the size of the browser side but now I don't have a good idea to avoid it.